example/node.go

```diff
@@ -0,0 +1,121 @@
+// Copyright © 2020 AMIS Technologies
```

Comment:
we can move send() and connect() to pm.go? only peer manager needs these two methods.

Reply:
I put these two functions in node.go because I think they are related to p2p.